frontend/src/scenes/insights/utils.tsx

```diff
@@ -322,6 +323,14 @@ export function formatBreakdownLabel(
     }
 
     if (breakdownFilter?.breakdown_type === 'cohort') {
+        if (breakdown_value === 'all' || breakdown_value === 0) {
```

Comment:
Adding this logic here which is from `getCohortNameFromId`.
i'm not sure what the `itemLabel` here would be in such cases, so I thought it would be better we stick to the original behavior which is to set as `All Users`.